### PR TITLE
Add a file for Eau Claire, WI

### PR DESCRIPTION
**Pull request by vpierson**
Purpose: add a hometown file for Eau Claire, WI

Questions:
1. If you've been to Eau Claire, what did you do? What did you think of it?
1. Have you been to the university?
1. What kind of food do you like?

### CODE REVIEW

File: vpierson-eauclaire.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+- Mona Lisa's
+- Tokyo's
+- Egg Roll Plus
+- Cancun
+### Shopping
+- Downtown shops
+- Oakwood Mall
+
+### Universities/Colleges
+- UWEC
```
Comment (SavageSpaceLion) [inline vpierson-eauclaire.md:15]: **I think I've been here before!** :thinking:

File: vpierson-eauclaire.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Eau Claire, WI
+
+### Restaurants
+- Mona Lisa's
+- Tokyo's
+- Egg Roll Plus
```
Comment (SavageSpaceLion) [inline vpierson-eauclaire.md]: Eggrolls sounds soooo gooood right now! Do they deliver :interrobang:

**Comment by SavageSpaceLion** [comment]
I've been to Eau Claire because my Grandmother lived there. I thought it was a pretty cool town. :sunglasses: 
I went to the university once... in the rain... to walk around the track. :cloud_with_rain: 
I cannot wait to have these eggrolls that were mentioned :exclamation: . I really like all food. :ramen:

**Comment by vpierson** [comment]
This pull request resolves #21

**Review by SavageSpaceLion — APPROVED** [review]
Looks great :exclamation: :rocket: :full_moon_with_face:

**Pull request merged**